package.json

```diff
@@ -81,7 +81,7 @@
     "enzyme": "3.3.0",
     "enzyme-adapter-react-16": "1.1.1",
     "enzyme-to-json": "3.1.2",
-    "eslint": "3.19.0",
+    "eslint": "5.16.0",
```

Comment:
As we discussed, it's impossible you update a dependency without changing `package-lock.json`. If something is not clear about this point, feel free to ping me.

Reply:
Why do we need to update our linter?

Reply:
No, this shouldn't be part of this pull request, my bad. This has already been fixed with the help of David on Friday, doesn't need to change now. Can we remove this change?